files/grest/rpc/01_cached_tables/stake_distribution_cache.sql

```diff
@@ -28,7 +28,7 @@ BEGIN
       AND block_no IS NOT NULL
     ORDER BY block_no DESC LIMIT 1;
 
-  SELECT MAX(id) INTO _last_account_tx_id FROM PUBLIC.TX WHERE block_id = _last_active_stake_blockid; 
+  SELECT MAX(id) INTO _last_account_tx_id FROM PUBLIC.TX WHERE block_id <= _last_active_stake_blockid; 
```

Comment:
This is for an edge case where the last block of the epoch has no transactions and the variable `_last_account_tx_id` becomes `NULL`. However, adding `<=` has worse performance. Could be better optimised.